client/protocols/backup.py

```diff
@@ -176,4 +188,21 @@ def send_messages(self, access_token, messages, timeout):
         response = urllib.request.urlopen(request, serialized_data, timeout)
         return json.loads(response.read().decode('utf-8'))
 
+    @staticmethod
+    def cp(msg):
```

Comment:
Is this the right place for this function? I'm not sure if there's a better place - but it's worth thinking about.